test/utils/mochaHooks.js

```diff
@@ -75,14 +115,13 @@ function createUnexpectedConsoleMessagesHooks(Mocha, methodName, expectedMatcher
       error.stack = '';
 
       if (isKarma) {
-        const testPath = `"${this.currentTest.parent
-          .titlePath()
-          .concat(this.currentTest.title)
-          .join('" -> "')}"`;
+        // @ts-expect-error -- this.currentTest being undefined would be a bug
+        const testPath = `"${this.currentTest.fullTitle()}"`;
```

Comment:
drive-by refactoring. Didn't know at the time that `fullTitle` existed. It's a bit different to the previous formatting but we use `fullTitle` in other parts as well so let's not surprise testers with new formatting.